instant-xml-macros/src/de.rs

```diff
@@ -58,16 +59,26 @@ impl Deserializer {
                 match data.fields {
                     syn::Fields::Named(ref fields) => {
                         fields.named.iter().enumerate().for_each(|(index, field)| {
-                            let is_element;
-                            let tokens = match retrieve_attr("attribute", &field.attrs) {
-                                Some(true) => {
-                                    is_element = false;
-                                    &mut attributes_tokens
+                            let mut field_namespace = None;
+                            let (tokens, def_prefix, is_element) = match retrieve_field_attribute(field) {
+                                Some(FieldAttribute::Namespace(value)) => {
+                                    field_namespace = Some(value);
+                                    (&mut elements_tokens, None, true)
                                 }
-                                _ => {
-                                    is_element = true;
-                                    &mut elements_tokens
+                                Some(FieldAttribute::PrefixIdentifier(def_prefix)) => {
+                                    if other_namespaces.get(&def_prefix).is_none() {
+                                        panic!("Namespace with such prefix do not exist for this struct");
```

Comment:
So this should yield an error, right?

Reply:
Actually, this happens in compile time, that's why it is `panic!`